chapter_1/ex1.rb

```diff
@@ -0,0 +1,7 @@
+puts "Hello World!"
+puts "Hello Again"
+puts "I like typing this"
+puts "This is fun."
+puts "Yay! Printing."
+puts "I'd much rather you 'not'."
```

Comment:
Why does Rubocop not complain about "Prefer single-quoted strings" on lines 6 and 7?